### PR TITLE
👷 use devflow to merge main into staging

### PR DESCRIPTION
**Pull request by thomas-lebeau**
## Motivation

Use [devflow](https://datadoghq.atlassian.net/wiki/spaces/DEVX/pages/2905376710/Devflow) to merge `main` back into `staging-XX`. 
If this fails, it will create a [fix PR](https://github.com/DataDog/browser-sdk/pull/2916) and log something like this:

```
couldn't merge the [main](https://github.com/DataDog/browser-sdk/tree/main) branch of [browser-sdk](https://github.com/DataDog/browser-sdk) into [thomas.lebeau/test-devflow-conflic](https://github.com/DataDog/browser-sdk/tree/thomas.lebeau/test-devflow-conflic), there was a conflict...

We have created a conflict resolution PR for you [#2916](https://github.com/DataDog/browser-sdk/pull/2916)
Alternatively, you can also reset the integration branch by using the following slack command: `/devflow reset-branch -r browser-sdk -b thomas.lebeau/test-devflow-conflic`

To get help about command usage, write `update-branch --help`
```

## Testing

- [x] Local
- [ ] Staging
- [ ] Unit
- [ ] End to end

---

I have gone over the [contributing](https://github.com/DataDog/browser-sdk/blob/main/CONTRIBUTING.md) documentation.

### CODE REVIEW

File: scripts/staging-ci/merge-into-staging.js [hunk 1]
```diff
@@ -1,55 +1,34 @@
...
+  const jsonResponse = await rawResponse.json()
+
+  printLog(jsonResponse)
+
+  const isSuccess = jsonResponse.state.feedbacks[0].level === SUCESS_FEEDBACK_LEVEL
+  const message = jsonResponse.state.feedbacks[0].message
+
+  if (!isSuccess) {
+    printError(message)
+    throw new Error(message)
```
Comment (BenoitZugmeyer) [inline scripts/staging-ci/merge-into-staging.js:30]: **💬 suggestion:** ‏No need to print the error here, the thrown error message will be displayed by `runMain`

File: scripts/staging-ci/merge-into-staging.js [hunk 2]
```diff
@@ -1,55 +1,34 @@
...
+    `${DEVFLOW_API_URL}/update-branch?repository=${REPOSITORY}&branch=${CURRENT_STAGING}`,
+    {
+      headers: {
+        Authorization: `Bearer ${DEVFLOW_AUTH_TOKEN}`,
+      },
+    }
+  )
+  const jsonResponse = await rawResponse.json()
+
+  printLog(jsonResponse)
```
Comment (BenoitZugmeyer) [inline scripts/staging-ci/merge-into-staging.js:23]: **❓ question:** ‏are you leaving this printLog for debug purpose?

Comment (thomas-lebeau) [reply scripts/staging-ci/merge-into-staging.js:23]: oops, I wanted to remove it!

**Review by BenoitZugmeyer — APPROVED** [review]

**Review by cy-moi — APPROVED** [review]
🚀🚀🚀

**Pull request merged**